File: PHP/PHP #37-42/09.php
<?php
$help_num = 3;
$nums = [4, 5, 6, 1, 2, 3];
$names = ["Ahmed", "Sayed", "Osama", "Mahmoud", "Gamal"] ;

for ($value = --$nums[$help_num] ; $value < sizeof($names) ; $value ++) :

    // echo ($names[$value] === "Sayed") ? $names[$value] . "\n" : ($names [$value] === "Osama") ? $names[$value] . "\n": null ;

endfor ;

?>

File: cpp/C++ #38-46/09.cpp
#include <iostream>
#include <array>
using namespace std ;

int main(){

    array<int,6> nums ;

    nums= {10, 20, 30, 40, 20, 50};
    
    cout << sizeof(nums) / sizeof(int) << "\n";
    cout << nums.size() << "\n";
    

    return 0;
}